code/src/java/pcgen/io/Compatibility.java

```diff
@@ -112,15 +122,15 @@ else if (!max.equals(min))
 			return null;
 		}
 		int closeParenLoc = first.lastIndexOf(')');
-		if (openParenLoc == -1)
+		if (closeParenLoc == -1)
```

Comment:
Interesting that was missed originally.  This was originally coded for the 5.12->5.14 conversion, so probably not really used anymore